test/fail_compilation/cppmangle.d

```diff
@@ -1,23 +1,10 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/cppmangle.d(10): Error: invalid zero length C++ namespace
-fail_compilation/cppmangle.d(14): Error: expected valid identifer for C++ namespace but got `0num`
-fail_compilation/cppmangle.d(18): Error: string expected following `,` for C++ namespace, not `)`
+fail_compilation/cppmangle.d(10): Error: string expected following `,` for C++ namespace, not `)`
```

Comment:
You need to update the line number and message, from the autotester:
```
expected:
----
fail_compilation/cppmangle.d(10): Error: string expected following `,` for C++ namespace, not `)`
----
actual:
----
fail_compilation/cppmangle.d(8): Error: found `,` when expecting `)`
fail_compilation/cppmangle.d(8): Error: declaration expected, not `)`
----
```
so this would become
```
fail_compilation/cppmangle.d(9): Error: found `,` when expecting `)`
fail_compilation/cppmangle.d(9): Error: declaration expected, not `)`
```